src/hearings/containers/request-hearing/date-priority-hearing/date-priority-hearing.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { GovUiConfigModel } from '@hmcts/rpx-xui-common-lib/lib/gov-ui/models';
+import { RefDataModel } from 'api/hearings/models/refData.model';
```

Comment:
copy api/hearings/models/refData.model into src/hearings/models/refData.model, node models and ng models need to be separated.